src/components/RecipeEdit.vue

```diff
@@ -461,6 +471,19 @@ beforeRouteLeave (to, from, next) {
     padding: 1rem;
 }
 
+.overlay {
+    display: block;
+    position: absolute;
+    width: 100%;
+    height: 100%;
```

Comment:
```suggestion
    height: 100%;
    left: 0px;
    top: 0px;
```

This should avoid intermediate showing of overflow bars of the browser.

Reply:
added